pkg/daemon/daemon.go

```diff
@@ -1170,8 +1196,8 @@ func (p *ptpProcess) cmdRun(stdoutToSocket bool, pm *PluginManager) {
 			break
 		} else {
 			glog.Infof("Recreating %s...", p.name)
-			newCmd := exec.Command(p.cmd.Args[0], p.cmd.Args[1:]...)
-			p.cmd = newCmd
+			newCmd := exec.Command(cmd.Args[0], cmd.Args[1:]...)
+			cmd = newCmd
```

Comment:
Shouldn't this be updating the struct?

Reply:
shouldn't this be doing p.cmd = newCmd? 
Does the process struct get properly updated by doing this?

Reply:
My worry is that if the process does die we will lose our handle on the running cmd and not being able to kill it when calling cmdStop.